neubot/__main__.py

```diff
@@ -183,9 +150,10 @@ def main(args):
         HTTP_SERVER.register_child(server, "/sapi")
 
     # Probe existing modules and ask them to attach to us
-    utils_modules.modprobe(None, "server", {
+    utils_modules.modprobe(ROOTDIR + "/neubot", None, "server", {
```

Comment:
This should better be written using `os.path.sep` for generality. However I'm not going to delay the merging process because of this, since the server is written to work on a Unix-like system.